yb-voyager/src/dbzm/dbzm.go

```diff
@@ -64,12 +75,111 @@ func findDebeziumDistribution() error {
 	return nil
 }
 
+func GenerateAndStoreYBStreamID(config *Config) (string, error) {
+	err := findDebeziumDistribution(config.SourceDBType)
+	if err != nil {
+		return "", fmt.Errorf("error in finding debezium distribution: %s", err)
+	}
+	YB_CLIENT_WRAPPER_JAR := fmt.Sprintf("%s/yb-client-cdc-stream-wrapper.jar", DEBEZIUM_DIST_DIR) //$DEBEZIUM_DIST/yb-client-cdc-stream-wrapper.jar
+	tableName := strings.Split(config.TableList[0], ".")[1]                                        //any table name in the database is required by yb-client createCDCStream(...) API
```

Comment:
Are we sure that the elements in the `config.TableList` are always qualified by the schema name?

How would the server find the table if it is in non-public schema and the table-name is not qualified?

Nit: avoid having arbitrary long lines. You can always put a comment just above the line it is describing. e.g.
```
// Any table name in the database is required by yb-client createCDCStream(...) API.
tableName := strings.Split(config.TableList[0], ".")[1]
```

Reply:
> How would the server find the table if it is in non-public schema and the table-name is not qualified?

For this, on the yb-client wrapper code side what is happening is we are doing a rest call to the server to get the list of tables and then we are the fetching the one with this tableName to get the object of some YBTable struct and only that object can be passed to createCDCStream() API as the table. ([refer](https://github.com/yugabyte/debezium/pull/72/files#diff-7bd741feeba6787d8e87e9380af921ff8a3e4ccc3d4841fde2080bd93bc817f5)) 
But I see a point to specify the schema name as well to jar and match the qualified name. Will make that change.